ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -797,3 +797,34 @@ def test_torch_logsumexp(
         dim=axis,
         keepdim=keepdims,
     )
+
+    
+@handle_frontend_test(
+    fn_tree="torch.mode",
+    dtype_and_x=statistical_dtype_values(
+        function="mode",
+        min_value=-1e04,
+        max_value=1e04,
+    ),
+    keepdims=st.booleans(),
+)
+def test_torch_mode(
+    *,
+    dtype_and_x,
+    keepdims,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    input_dtype, x, axis = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        dim=axis,
+        keepdim=keepdims,
+    )

```

Comment:
The code seems alright, but the tests are failing so there must be some bug you'll need to solve. You can see the error in the intelligent-tests-pr/run_tests_1 CI checks. I would suggest you run the test locally to debug it.

Reply:
Yeah I already checked the error it was throwing pynvml warning but I already Installed it.
![Screenshot 2023-04-11 000454](https://user-images.githubusercontent.com/90236987/230969708-a295cc8d-f811-4c44-8c0c-b051c91f9b25.png)

Reply:
How about this 
![image](https://user-images.githubusercontent.com/111577222/230994472-7888dcc6-0e19-483e-9bed-264c17c52d15.png)